packages/integration_test/README.md

```diff
@@ -109,7 +109,7 @@ This feature is currently supported on Android, iOS, and Web.
 ```dart
 void main() {
   final IntegrationTestWidgetsFlutterBinding binding =
-      IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+      IntegrationTestWidgetsFlutterBinding();
```

Comment:
```suggestion
      IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

Reply:
Also this cast the type declaration can be removed:

```dart
final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

Reply:
Have updated it